src/app/assess/result/summary/summary-header/summary-header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="flex flexItemsCenter" id="assessmentsSummaryHeaderComponent">
+  <div class="headerRow" id="bannerRow">
+    <span>Good news!</span>
+  </div>
+  <div class="headerRow">
+    <span>We've reduced your risk already,</span>
+  </div>
+  <div class="headerRow">
+    <span>but we can improve some areas of your defense.</span>
+  </div>
+</div>
```

Comment:
Do we want to turn this tslint warning off?

Reply:
Doesn't really matter to me. I always add the newline for TS files because of the warning.
I guess I should add it to other types of files too; if we're going to keep it.